#Start(1688442595142)

  We've found a bug for you!
  /home/trite/git/organizational-app/webapp/src/App.res:28:5-31:37

  26 │ let make = async () => {
  27 │   let result =
  28 │     await PostgrestClient.postgrestClient("http://localhost:3000")
  29 │     ->PostgrestClient.from("users")
  30 │     ->PostgrestQueryBuilder.select()
  31 │     ->PostgrestQueryBuilder.execute()
  32 │ 
  33 │   <div>

  Await on expression not in an async context

FAILED: cannot make progress due to previous errors.
#Done(1688442595154)
